source/concepts.tex

```diff
@@ -273,13 +273,19 @@
 \indexlibrary{\idxcode{Same}}%
 \begin{itemdecl}
 template<class T, class U>
-  concept Same = is_same_v<T, U>;
+  concept @\placeholdernc{same-impl}@ = // exposition only
+    is_same_v<T, U>;
```

Comment:
Could we please compact this onto a single line? I doubt the comment will be confusing; we have other places where a data member including initializer is exposition-only, and I think it's clear that the value of the initializer salient.